clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/agent/OpenstackServerGroupCachingAgent.groovy

```diff
@@ -286,6 +287,14 @@ class OpenstackServerGroupCachingAgent extends AbstractOpenstackCachingAgent imp
       result.put('minSize', stack.parameters?.get('min_size') ?: 0)
       result.put('maxSize', stack.parameters?.get('max_size') ?: 0)
       result.put('desiredSize', stack.parameters?.get('desired_size') ?: 0)
+      result.put('autoscalingType', ServerGroupParameters.AutoscalingType.fromMeter(stack.parameters.get('autoscaling_type')))
+      ['up','down'].each {
+        Map<String, Object> scaler = [cooldown  : stack.parameters.get("scale${it}_cooldown"?.toString()),
```

Comment:
I'm wondering why you have the `?.` operator on that string, I can't see it ever being `null`.